src/main/java/mate/academy/springboot/swagger/controller/ProductController.java

```diff
@@ -81,14 +81,16 @@ ProductResponseDto findById(@PathVariable
 
     @PostMapping
     @ApiOperation(value = "Create new product")
-    ProductResponseDto save(@RequestBody ProductRequestDto dto) {
+    ProductResponseDto save(@RequestBody(description =

```

Comment:
I meant the content schema part for describing the actual model, check the firs answer on that StackOverflow thread
https://stackoverflow.com/questions/64645528/java-spring-boot-openapi-3-how-to-add-description-for-requestbody